terraform/variables.tf

```diff
@@ -69,6 +69,11 @@ variable "validator_log_level" {
   default     = "debug"
 }
 
+variable "validator_linux_capacities" {
+  type        = list(string)
```

Comment:
Doesn't this need a default?

Reply:
Good catch! Set the default as empty list